### PR TITLE
Fixing develop by removing swipe module leaving it in core_ui instead

### PR DESCRIPTION
**Pull request by kclaudeeager**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -36,7 +36,10 @@ android {
         getByName("release") {
             isMinifyEnabled = true
             isDebuggable = true
-            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+            proguardFiles(
```
Comment (esthcarelle) [inline app/build.gradle.kts:39]: This should not be removed please

**Review by esthcarelle — CHANGES_REQUESTED** [review]

File: app/build.gradle.kts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 plugins {
     id("com.android.application")
-    id ("org.jetbrains.kotlin.android")
-    id ("com.google.firebase.appdistribution")
+    id("org.jetbrains.kotlin.android")
+    id("com.google.firebase.appdistribution")
```
Comment (bensek) [inline app/build.gradle.kts:4]: Format these

File: app/build.gradle.kts [hunk 3]
```diff
@@ -85,25 +85,16 @@ dependencies {
...
     implementation(project(Modules.dataOnboarding))
     implementation(project(Modules.domainOnboarding))
     implementation(project(Modules.uiOnboarding))
     implementation(project(Modules.coreUI))
-
-    implementation(project(Modules.uiSwipe))
-
     implementation(project(Modules.uiAdmin))
-
     implementation(AndroidX.coreKtx)
```
Comment (bensek) [inline app/build.gradle.kts:98]: Add back these lines. This change was not necessary.

File: app/build.gradle.kts (hunk 2 above)
Comment (bensek) [reply app/build.gradle.kts:4]: Please fix this and add back the spaces after id, it is going to cause conflicts to other people unnecessarily @kclaudeeager

Comment (kclaudeeager) [reply app/build.gradle.kts]: done

**Review by bensek — APPROVED** [review]

**Review by esthcarelle — APPROVED** [review]

**Pull request merged**